exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/_metric_exporter/__init__.py

```diff
@@ -80,7 +82,7 @@ def _translate_data(
         self, data: Sequence[Metric]
     ) -> ExportMetricsServiceRequest:
         sdk_resource_instrumentation_library_metrics = {}
-        self._collector_metric_kwargs = {}
+        self._collector_kwargs = {}
```

Comment:
I think you could do the same thing here with `pb2.Metric` and set the fields below instead of `pb2.Metric(**self._collector_kwargs)` right?

Reply:
updated the code.